realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -444,7 +444,7 @@ public SyncUser run() {
      * @return {@code SyncUser} associated with the given identity provider and providerId, or {@code null} in case
      * of an {@code invalid} provider or {@code providerId}.
      * @throws ObjectServerError in case of an error.
-     * @deprecated  As of release 3.6.0, replaced by {@link #retrieveInfoForUser(String, String)}}
+     * @deprecated  as of release 3.6.0, replaced by {@link #retrieveInfoForUser(String, String)}}
```

Comment:
It seems two spaces between `@deprecated` and `as`. Can we use one space instead?